README.md

```diff
@@ -72,4 +38,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] Your license link is broken. Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/) . :smile: